infra/build/functions/build_project.py

```diff
@@ -180,6 +184,12 @@ def get_datetime_now():
   return datetime.datetime.now()
 
 
+def get_datetime_yesterday():
```

Comment:
Rather than this, we could also try to figure out the actual latest uploaded report by looking at the bucket. Can leave this as a TODO.

Reply:
yes that is what we have to change later. For now I changed it to use the current date. It means it needs the coverage build have to finish before starting introspector build.